source/manual/aws-backup-and-restore-introduction.html.md

```diff
@@ -0,0 +1,23 @@
+---
+owner_slack: "#govuk-infrastructure"
+title: Backup and restore databases in RDS - introduction
+section: AWS
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2017-12-05
+review_in: 1 month
+---
+
+# Backup and restore databases in AWS - Introduction
+
+In AWS, we have five datastores: MySQL, PostgreSQL, ElasticSearch, MongoDB, and
+Redis. Whether we back them up, restore them, and how we do so depends on how
```

Comment:
This sentence isn't really needed. Details about how they're managed can go below in short summary sections.